common/src/main/java/com/wynntils/models/players/PlayerModel.java

```diff
@@ -28,10 +29,17 @@
 
 public final class PlayerModel extends Model {
     private static final Pattern GHOST_WORLD_PATTERN = Pattern.compile("^_(\\d+)$");
+    private static final int MAX_ERRORS = 5;
 
     private final Map<UUID, WynntilsUser> users = new ConcurrentHashMap<>();
     private final Set<UUID> fetching = ConcurrentHashMap.newKeySet();
     private final Map<UUID, Integer> ghosts = new ConcurrentHashMap<>();
+    private int errorCount;
+
+    @Override
+    public void init() {
+        errorCount = 0;
```

Comment:
/wynntils reauth should clear this too, not /reload only

Reply:
Good point. I added it for the TerritoryManager as well.